packages/odyssey-react-mui/src/OdysseyCacheProvider.test.tsx

```diff
@@ -11,6 +11,7 @@
  */
 
 import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
```

Comment:
Why do we need this import here?

Reply:
Ugh, this. For some reason, _not_ having it breaks our typechecks. https://github.com/testing-library/jest-native/issues/136

Reply:
https://stackoverflow.com/questions/57861187/property-tobeinthedocument-does-not-exist-on-type-matchersany